run/do-person-harvest.sh

```diff
@@ -45,8 +45,19 @@ illiad_date=`/s/sirsi/Unicorn/Bin/transdate -d-1`
 echo "Updating/Inserting keys from /s/SUL/Batchlog/userload.keys.$illiad_date into ILLiad" >> $LOG/harvest.log
 $HOME/run/pop2illiad.sh $illiad_date
 
+# Run harvest.xml.out through folio_api_client ruby script to load users into FOLIO
+cd $FOLIO/lib
+# Split into batches of 5000
+while mapfile -t -n 5000 array && ((${#array[@]}))
+do
+    printf '%s\n' "${array[@]}" > tmp.xml
```

Comment:
should be $OUT/tmp.xml here?

Reply:
and on line 55?

Reply:
guess it’s just a temporary file, but seems better to have it exist somewhere other than the lib directory?  Don’t feel too strongly

Reply:
Good catch. Pushed another commit.